package.json

```diff
@@ -17,7 +17,14 @@
     "grunt-contrib-jshint": "~0.6.3",
     "grunt-contrib-nodeunit": "~0.2.0",
     "grunt-contrib-uglify": "~0.2.2",
-    "habitat": "~1.1.0"
+    "habitat": "~1.1.0",
+    "babel-core": "^6.5.2",
+    "babel-loader": "^6.2.3",
+    "babel-plugin-react-transform": "^2.0.2",
+    "babel-preset-es2015": "^6.5.0",
+    "babel-preset-react": "^6.5.0",
+    "babel-preset-stage-0": "^6.5.0",
+    "webpack": "^1.13.1"
```

Comment:
If we're adding this much stuff, please also add an npm script entry so that `npm run build` for instance will run the webpack command.